io/js/src/test/scala/fs2/io/IoPlatformSuite.scala

```diff
@@ -108,7 +108,8 @@ class IoPlatformSuite extends Fs2Suite {
     }
 
     Stream
-      .empty[IO]
+      .empty
+      .covary[IO]
```

Comment:
Out of curiosity, why this change?

Reply:
Oh, sorry, I missed https://github.com/typelevel/fs2/pull/3242#discussion_r1241253824. Thanks!

Reply:
There's a [compile error in the build](https://github.com/typelevel/fs2/actions/runs/5371040241/jobs/9743532919#step:10:44) (locally too), which I'm still figuring out:
```
[error] /home/runner/work/fs2/fs2/io/js/src/test/scala/fs2/io/IoPlatformSuite.scala:111:13: value empty of type fs2.Stream[fs2.Pure,Nothing] does not take type parameters.
[error]       .empty[IO]
[error]             ^
[error] one error found
```
It's caused by the implicit added for the pretty printer.

I _think_ it's interfering with the resolution of the `monoidKInstance`, that provides the `Stream.empty[IO]` function:
```
  implicit def monoidKInstance[F[_]]: MonoidK[Stream[F, *]] =
    new MonoidK[Stream[F, *]] {
      def empty[A]: Stream[F, A] = Stream.empty
      def combineK[A](x: Stream[F, A], y: Stream[F, A]): Stream[F, A] = x ++ y
    }
```

But I don't know why this would be the case.

Reply:
I think I know why: 

`Stream.empty[IO]` uses an `apply`, so is desugared to `Stream.empty.apply[IO]`:
 
The `apply[F[_]]` function is added by `PureOps`:

```scala
  implicit final class PureOps[O](private val self: Stream[Pure, O]) extends AnyVal {
    def apply[F[_]]: Stream[F, O] = covary
}
```
This conflicts with [the `apply` function in the scalacheck pretty printer](https://github.com/typelevel/scalacheck/blob/6df98d611ae6c2cf584596ac18628494c6dfac79/core/shared/src/main/scala/org/scalacheck/util/Pretty.scala#L19).